build/common/installer/scripts/tomlparser-common-agent-config.rb

```diff
@@ -9,6 +9,7 @@
 @configSchemaVersion = ""
 
 @disableTelemetry = false
+@logEnableKubernetesMetadataCacheTTL = "60"
```

Comment:
1. have the seconds suffix for the variable name, i.e.  @logEnableKubernetesMetadataCacheTTLSeconds
2. use the integer instead of string.

Reply:
changed